src/components/calcite-radio-button/calcite-radio-button.tsx

```diff
@@ -123,26 +124,35 @@ export class CalciteRadioButton {
 
   private titleAttributeObserver: MutationObserver;
 
+  /** @internal */
+  @Method()
```

Comment:
I think you just need to setup a register event that is emitted on `connectedCallback` that then calls `checkLastRadioButton()`?

Reply:
Just tried doing something like this in radio-button, but it doesn't always work.  Some times when I load the page it works, but other times, no checked value results.  I suspect this is a race condition based on the random order that all radios in a group are registered.  I discovered this random order by logging to the console the value of each radio from inside `componentDidLoad`.

```
@Listen("calciteRadioButtonRegister")
 registerHandler():void {
   if (this.name) {
     this.checkLastRadioButton();
   }
 }
```

Reply:
The random-order thing always happens with sibling elements - might be relevant: https://github.com/Esri/calcite-components/blob/d64982fb461d35a835e7e2f6a52a0e684b584f8f/src/components/calcite-dropdown-item/calcite-dropdown-item.tsx#L266